src/ProtocolFees.sol

```diff
@@ -59,7 +59,7 @@ abstract contract ProtocolFees is IProtocolFees, Owned {
     /// @dev to prevent an invalid protocol fee controller from blocking pools from being initialized
     ///      the success of this function is NOT checked on initialize and if the call fails, the protocol fees are set to 0.
     /// @dev the success of this function must be checked when called in setProtocolFee
-    function _fetchProtocolFee(PoolKey memory key) internal returns (bool success, uint24 protocolFees) {
+    function _fetchProtocolFee(PoolKey memory key) internal returns (bool success, uint24 protocolFee) {
```

Comment:
so i belieeeeve it was plural because it contains 2 fees, one for each direction. But I do agree with this change it sounds better

Reply:
ohh i see. we could do `_fetchProtocolFees`? they should either be both singular or both plural